src/common/bibliography.F

```diff
@@ -4886,6 +4886,86 @@ SUBROUTINE add_all_references()
                          "ER"), &
                          DOI="10.48550/arXiv.2306.16066")
 
+      CALL add_reference(key=Han2018, ISI_record=s2a( &
```

Comment:
Why doesn't it call `Wang2018`

Reply:
Sorry I don't understand your meaning. I mean the key should be last name + year instead of first name + year.

Reply:
Ah, sorry for the misunderstanding.